### PR TITLE
Feature/exchange fiat selector horizontal layout

### PR DESCRIPTION
**Pull request by poocart**
Includes improvements to make Exchange fiat select horizontal as it is in People search.

### CODE REVIEW

File: src/components/SelectorInput/SelectorInput.js [hunk 1]
```diff
@@ -295,16 +389,23 @@ export default class SelectorInput extends React.Component<Props, State> {
         || (name && name.toUpperCase().includes(query.toUpperCase())))
       : options;
 
+    const filteredHorizontalListData = (query && query.length >= MIN_QUERY_LENGTH && horizontalOptions.length)
```
Comment (jsidorenko) [inline src/components/SelectorInput/SelectorInput.js]: You're becoming master of complex and unreadable code :D

File: src/screens/Exchange/Exchange.js [hunk 2]
```diff
@@ -811,6 +817,21 @@ class ExchangeScreen extends React.Component<Props, State> {
     });
   };
 
+  generateFiatOptions = () => {
+    return fiatCurrencies.map(({ symbol, iconUrl, ...rest }) => {
+      return ({
```
Comment (jsidorenko) [inline src/screens/Exchange/Exchange.js]: No need to wrap with `()` here

File: src/components/SelectorInput/SelectorInput.js (hunk 1 above)
Comment (jsidorenko) [reply src/components/SelectorInput/SelectorInput.js]:
```
let filteredHorizontalListData = horizontalOptions;
if (query && query.length >= MIN_QUERY_LENGTH && horizontalOptions.length) {
  filteredHorizontalListData = horizontalOptions.filter(({ value: val, name }) => {
    val = val ? val.toUpperCase() : '';
    name = name ? name.toUpperCase() : '';
    const searchQuery = query.toUpperCase();
    return val.includes(searchQuery) || name.includes(searchQuery);
  });
}
```

Comment (poocart) [reply src/components/SelectorInput/SelectorInput.js]: Thanks, improved in new commit 👍

File: src/screens/Exchange/Exchange.js (hunk 2 above)
Comment (poocart) [reply src/screens/Exchange/Exchange.js]: Thanks, improved in new commit 👍

File: src/components/SelectorInput/SelectorInput.js [hunk 3]
```diff
@@ -386,17 +386,19 @@ export default class SelectorInput extends React.Component<Props, State> {
...
-        || !isSearchQuery,
-    );
+    const isSearchQuery = query && query.length >= MIN_QUERY_LENGTH;
+
+    let filteredListData = options;
+    let filteredHorizontalListData = horizontalOptions;
+
+    if (isSearchQuery) {
+      filteredListData = filteredListData.filter(
+        ({ value: val, name }) => isMatchingSearch(query, val) || isMatchingSearch(query, name),
```
Comment (jsidorenko) [inline src/components/SelectorInput/SelectorInput.js]: Perfecto!

Comment (poocart) [reply src/components/SelectorInput/SelectorInput.js]: ¡muchas gracias Señor!

**Pull request merged**